engine/schema/src/main/resources/META-INF/db/schema-41600to41610.sql

```diff
@@ -0,0 +1,23 @@
+-- Licensed to the Apache Software Foundation (ASF) under one
+-- or more contributor license agreements.  See the NOTICE file
+-- distributed with this work for additional information
+-- regarding copyright ownership.  The ASF licenses this file
+-- to you under the Apache License, Version 2.0 (the
+-- "License"); you may not use this file except in compliance
+-- with the License.  You may obtain a copy of the License at
+--
+--   http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing,
+-- software distributed under the License is distributed on an
+-- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+-- KIND, either express or implied.  See the License for the
+-- specific language governing permissions and limitations
+-- under the License.
+
+--;
+-- Schema upgrade from 4.16.0.0 to 4.16.1.0
+--;
+
+-- Change the type of the 'ipsec_psk' field to allow large PSK.
```

Comment:
CLGTM,

Just one point to raise, I think it would be good to inform the PR number here.

Reply:
Done.
Thanks for the suggestion @GutoVeronezi